qml/main.qml

```diff
@@ -40,6 +40,143 @@ ApplicationWindow {
 
     cover: Qt.resolvedUrl("pages/CoverPage.qml")
 
+    ListModel {
+        id: regions
+
+        ListElement {
+            name: QT_TR_NOOP("Finland")
+            apiName: "finland"
+        }
+        ListElement {
+            name: QT_TR_NOOP("Helsinki and the capital region (HRT)")
+            identifier: "helsinki"
+            apiName: "hsl"
+            boundarycirclelat: 60.169
+            boundarycirclelon: 24.940
+        }
+        ListElement {
+            name: QT_TR_NOOP("Tampere region (Nysse)")
+            identifier: "tampere"
+            apiName: "finland"
+            boundarycirclelat: 61.498
+            boundarycirclelon: 23.759
+        }
+        ListElement {
+            name: QT_TR_NOOP("Turku region (Föli)")
+            identifier: "turku"
+            apiName: "finland"
+            boundarycirclelat: 60.451
+            boundarycirclelon: 22.267
+        }
+        ListElement {
+            name: QT_TR_NOOP("Hämeenlinna region")
+            identifier: "hameenlinna"
```

Comment:
Hämeenlinna seems to have at least one very long, UI breaking, line "number" `4 Inter Kuljetus Oy` but I think we cannot fix that easily.